scripts/protein/esm2/esm2_pretrain.py

```diff
@@ -380,6 +381,19 @@ def main(
     default=10000,
     help="Number of steps between validation. Default is 10000.",
 )
+parser.add_argument(
+    "--log-every-n-steps",
+    type=int,
+    required=False,
+    help="Number of steps between logging. Default is 50.",
+)
+parser.add_argument(
+    "--save-every-n-steps",
+    type=int,
+    required=False,
+    default=10000,
+    help="Number of steps between saving checkpoints. Default is 10000.",
+)
```

Comment:
Generally I think the val_check_interval is a good default for `save-ever-n-steps` since then when you go through the process of validation, you also save the checkpoint if it's worth saving. Otherwise why are we pausing to validate if we don't have a record of the model? I'd personally prefer getting rid of this and having val_interval == save_interval. Log interval is great though!